### ISSUE TITLE
Test suite compilation failure

### ISSUE THREAD

**Issue by snoyberg** [alcidesv/second-transfer#1]
```
Linking dist/build/compiling-ok/compiling-ok ...
/tmp/stackage-build1/second-transfer-0.5.3.2/dist/build/libHSsecond-transfer-0.5.3.2-D7sUxN7OlNr3XtJwDRcgoD.a(tlsinc.o):(.text+0x304): undefined reference to `SSL_CTX_set_ecdh_auto'
/tmp/stackage-build1/second-transfer-0.5.3.2/dist/build/libHSsecond-transfer-0.5.3.2-D7sUxN7OlNr3XtJwDRcgoD.a(tlsinc.o):(.text+0x340): undefined reference to `SSL_CTX_set_alpn_select_cb'
/tmp/stackage-build1/second-transfer-0.5.3.2/dist/build/libHSsecond-transfer-0.5.3.2-D7sUxN7OlNr3XtJwDRcgoD.a(tlsinc.o): In function `wait_for_connection':
(.text+0xa01): undefined reference to `SSL_get0_alpn_selected'
collect2: error: ld returned 1 exit status
```

**Comment by alcidesv** [comment]
This is probably happening because I'm using OpenSSL 1.0.2 ... unfortunately older versions don't have ALPN. Besides [this one](https://github.com/fpco/stackage/commit/cc5b1ff00fdbd89a2d6f378841f3ff9ab8e40d85), would it be desirable to have some other work-around?

**Comment by snoyberg** [comment]
I guess it's your call. As it stands, Stackage won't be able to test regressions in the library.

**Comment by DanBurton** [comment]
Bump. Although I've [re-enabled the test suite for second-transfer](https://github.com/fpco/stackage/commit/5d8ed0dd3a35a856c480d0c75dbc8280ab282a98), it's still [listed as an expected failure](https://github.com/fpco/stackage/blob/5d8ed0dd3a35a856c480d0c75dbc8280ab282a98/build-constraints.yaml#L1903).

**Comment by alcidesv** [comment]
Thanks for the heads-up, I will take a look.

**Comment by alcidesv** [comment]
This issue is closed by 8b8afa1ef
